llvm-build/llvm.cmake

```diff
@@ -72,4 +72,4 @@ else()
   set(LLVM_BUILD_LLVM_DYLIB ON CACHE BOOL "")
   set(CPACK_BINARY_TGZ ON CACHE BOOL "")
 endif()
-set(LLVM_DYLIB_COMPONENTS "core;debuginfodwarf;linker;support;target;bitwriter;analysis" CACHE STRING "")
+set(LLVM_DYLIB_COMPONENTS "core;debuginfodwarf;linker;support;target;bitwriter;analysis;executionengine;runtimedyld;mcjit;bitstreamreader;bitreader;native" CACHE STRING "")
```

Comment:
@swernli Might be good to add a permalink to the docs page that lists theses dep chains.

Reply:
Good idea, I'll add that!